src/cgame/cg_buildable.cpp

```diff
@@ -526,21 +526,21 @@ static bool CG_ParseBuildableSoundFile( const char *filename, buildable_t builda
 	return true;
 }
 
-static bool CG_RegisterBuildableAnimation( buildableInfo_t *ci, const char *modelName, int anim, const char *animName,
+static bool CG_RegisterBuildableAnimation( buildableInfo_t *ci, const char *modelName, const char* foundSuffix, int anim, const char *animName,
```

Comment:
How about a single argument with the contents of `fileBase` from the other function, instead of modelName+foundSuffix?

Reply:
Done.

I also changed the md5 animation file name for buildables in the process, so `fileBase` is now the same for both iqm and md5 models. Because of the file name difference there was a mistake in my initial code.

Before it was `models/spiker/attack.md5anim`, now  it's `models/spiker/spiker_attack.md5anim`.

For information this is what already does weapon models (player models still do the same way as buildable before), so this is a step forward path name unification and with a more robust naming scheme as you can copy/move the md5 model and the anim files without taking risk of overwriting files from another model.

We don't have md5 buildable models (all have been converted to IQM in the past) so this path name change doesn't require to modify assets.